### PR TITLE
Include /dev/infiniband

### PR DESCRIPTION
**Pull request by jmstover**
**Description of the Pull Request (PR):**

If `--contain` is used, then include the host `/dev/infiniband` if
it exists.

**Checkoff for all PRs:**

- [x] I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [x] I have added changes to the [CHANGELOG](https://github.com/singularityware/singularity/blob/development/CHANGELOG.md) and and documentation updates to the [singularityware](https://www.github.com/singularityware/singularityware.github.io) documentation base.
- [x] I have tested this PR locally with a `make test`
- [x] This PR is NOT against the project's master branch
- [] I have added myself as a contributor to the [contributors's file](https://github.com/singularityware/singularity/blob/master/CONTRIBUTORS.md)
- [x] This PR is ready for review and/or merge

Attn: @singularityware-admin

### CODE REVIEW

File: src/lib/runtime/mounts/dev/dev.c [hunk 1]
```diff
@@ -105,6 +105,20 @@ int _singularity_runtime_mount_dev(void) {
         bind_dev(sessiondir, "/dev/random");
         bind_dev(sessiondir, "/dev/urandom");
 
+        /* If /dev/infiniband exists include it */
+        if ( is_dir("/dev/infiniband") == 0 ) {
+            int ret;
```
Comment (GodloveD) [inline src/lib/runtime/mounts/dev/dev.c]: Doesn't look like this is being used anywhere.  Copy and paste mistake?

**Review by GodloveD — DISMISSED** [review]
Looks like there is an unused var being declared.

File: src/lib/runtime/mounts/dev/dev.c (hunk 1 above)
Comment (jmstover) [reply src/lib/runtime/mounts/dev/dev.c]: No ... I _was_ going to check the return value and check that in an `if` ... but then changed how I was checking and forgot to remove that.

**Review by cclerget — APPROVED** [review]
Looks good

**Comment by jmstover** [comment]
Re-open after branch removal

**Pull request closed without merging**